frame/contracts/src/chain_extension.rs

```diff
@@ -206,6 +210,22 @@ impl<'a, 'b, E: Ext, S: state::State> Environment<'a, 'b, E, S>
 where
 	<E::T as SysConfig>::AccountId: UncheckedFrom<<E::T as SysConfig>::Hash> + AsRef<[u8]>,
 {
+	/// The function id within the `id` passed by a contract.
+	///
+	/// It returns the two least significant bytes of the `id` passed by a contract as the other
+	/// two bytes represent the chain extension itself (the code which is calling this function).
+	pub fn func_id(&self) -> u16 {
+		(self.inner.id & 0x00FF) as u16
```

Comment:
Shouldn't it be 0xFFFF?

Reply:
Why? That would be a no-op then.

Reply:
Oops. You are right. This just masks the last byte. Do you are to open a PR? Would be good to modify one of the tests to actually catch that (change one of the function ids to use the second byte, too).